cmd/platform/user.go

```diff
@@ -86,26 +89,58 @@ var deleteAllUsersCmd = &cobra.Command{
 	RunE:    deleteAllUsersCommandF,
 }
 
-var migrateAuthCmd = &cobra.Command{
-	Use:   "migrate_auth [from_auth] [to_auth] [match_field]",
```

Comment:
Changing this command could potentially break someone's script. Could we keep the command the same and just document the different options for the different `to_auth` types?

Reply:
That's a good point. I'm not sure if this is the kind of command that de people use periodically, but anyway, there exists the possibility that we are breaking something.

I think the resulting "help" information will be more confusing, but if we don't want to break anything, is probably the best option.

Reply:
Ok, finally I merged again the commands and rewrite the help of the commands. Now I think the result is clear and the API hasn't changed (except for the extra --dryRun param in the ldap command).